README.md

```diff
@@ -237,6 +237,7 @@ If you are reading this it looks like you are looking to add an egg to your serv
 * [Subnautica: Nitrox Mod](game_eggs/steamcmd_servers/subnautica_nitrox_mod)
 * [Sven Co-op](game_eggs/steamcmd_servers/svencoop)
 * [The Forest](game_eggs/steamcmd_servers/the_forest)
+* [The Isle](game_eggs/steamcmd_servers/the_isle)
```

Comment:
If there are 2 versions of the game make a sub-folder

so `game_eggs/steamcmd_servers/the_isle/evrima` and `game_eggs/steamcmd_servers/the_isle/legacy`

Reply:
I've made the evrima sub folder, I'll make the legacy sub folder once I've double checked the egg.